api/src/pcapi/core/bookings/repository.py

```diff
@@ -232,7 +232,7 @@ def find_user_ids_with_expired_individual_bookings(expired_on: date | None = Non
         user_id
         for user_id, in (
             db.session.query(User.id)
-            .join(Booking)
+            .join(Booking, User.userBookings)
```

Comment:
Quel rapport avec le ticket ?

Reply:
sans ce code cela créer l'erreur 
FAILED tests/scripts/booking/handle_expired_bookings_test.py::NotifyUsersOfExpiredBookingsTest::should_notify_of_todays_expired_bookings - sqlalchemy.exc.AmbiguousForeignKeysError: Can't determine join between 'user' and 'booking'; tables have more than one foreign key constraint relationship between them. Please ...

d'ou cet ajout